File: Skee-Ball/Skee-Ball.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int sc = Int32.Parse(Console.ReadLine());
            int tic = Int32.Parse(Console.ReadLine());
            if((sc/12)>=tic)
            Console.WriteLine("Buy it!");
            else
            Console.WriteLine("Try again");
            
        }
    }
}

File: Skee-Ball/Skee-Ball.java
import java.util.Scanner;
public class Program
{
    public static void main(String[] args) {
    Scanner in = new Scanner(System.in);
    int sc = in.nextInt();
    int tic = in.nextInt();
    if((sc/12)>=tic)
    System.out.println("Buy it!");
    else
    System.out.println("Try again");
        
    }
}
